orchagent/port.h

```diff
@@ -164,6 +164,7 @@ class Port
     uint32_t  m_nat_zone_id = 0;
     uint32_t  m_vnid = VNID_NONE;
     uint32_t  m_fdb_count = 0;
+    uint32_t  m_flap_count = 0;
```

Comment:
@prgeor why not uint64? Do we expect an overflow?  Should be reflected in the log? Any reaction from the SWSS is needed?

Reply:
@nazariig done